lib/ansible/modules/cloud/docker/docker_host_info.py

```diff
@@ -245,15 +245,18 @@ def get_docker_items_list(self, docker_object=None, filters=None, verbose=False)
         header_images = ['Id', 'RepoTags', 'Created', 'Size']
         header_networks = ['Id', 'Driver', 'Name', 'Scope']
 
+        filter_arg = dict()
+        if filters:
+            filter_arg['filters'] = filters
```

Comment:
This change is purely for backward compatibility with docker-py? If yes what was wrong with the original approach?

Reply:
`client.networks()` only got support for the `filters` argument in docker-py 2.0.2; apparently it was forgotten to add it until then (the API probably supported it since the beginning, or at least much earlier). I could have also put this into the `networks` branch only (and leave the others as-is).